### PR TITLE
Use const in pdf_thumbnail_viewer.js

### PR DESCRIPTION
**Pull request by wojtekmaj**

### CODE REVIEW

File: web/pdf_thumbnail_viewer.js [hunk 1]
```diff
@@ -257,8 +258,8 @@ class PDFThumbnailViewer {
   }
 
   forceRendering() {
-    let visibleThumbs = this._getVisibleThumbs();
-    let thumbView = this.renderingQueue.getHighestPriority(visibleThumbs,
+    const visibleThumbs = this._getVisibleThumbs();
+    const thumbView = this.renderingQueue.getHighestPriority(visibleThumbs,
                                                            this._thumbnails,
                                                            this.scroll.down);
```
Comment (Snuffleupagus) [inline web/pdf_thumbnail_viewer.js]: Nit: Please fix the indentation here.

Comment (timvandermeij) [reply web/pdf_thumbnail_viewer.js]: Afterwards, please squash all commits into one as described at https://github.com/mozilla/pdf.js/wiki/Squashing-Commits so we can merge it. Thanks!

**Review by timvandermeij — APPROVED** [review]

**Comment by timvandermeij** [comment]
Thank you!

**Pull request merged**